### PR TITLE
making changes for 3.3.0 rc1

### PR DESCRIPTION
**Pull request by GodloveD**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,30 @@ _The old changelog can be found in the `release-2.6` branch_
 
 # Changes Since v3.2.1
 
+# v3.3.0 - [2019.06.17]
+
+## Changed defaults / behaviors
+
+  - Remote login and status commands will now use the default remote if a remote name is not supplied
+  - Added Singularity hub cache support when using the `pull` command
```
Comment (WestleyK) [inline CHANGELOG.md]: Mintpick... Maybe?

```suggestion
  - Added Singularity hub (`shub`) cache support when using the `pull` command
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,30 @@ _The old changelog can be found in the `release-2.6` branch_
...
+  - Clean cache in a safer way by only deleting subdirectories
```
Comment (WestleyK) [inline CHANGELOG.md]:
```suggestion
  - Clean cache in a safer way by only deleting the cache subdirectories
```

EDIT: something like that, you get the idea...

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,30 @@ _The old changelog can be found in the `release-2.6` branch_
...
 # Changes Since v3.2.1
 
+# v3.3.0 - [2019.06.17]
+
+## Changed defaults / behaviors
+
+  - Remote login and status commands will now use the default remote if a remote name is not supplied
+  - Added Singularity hub (`shub`) cache support when using the `pull` command
+  - Clean cache in a safer way by only deleting subdirectories
+  - Improvements to the `cache clean` command
```
Comment (WestleyK) [inline CHANGELOG.md:21]: Dosnt this line mean the same as the line right above it?

Comment (GodloveD) [reply CHANGELOG.md:21]: No.  Didn't you add shub caching and some other improvements?  I didn't know what all you changed so I just wrote this as a catch all.

Comment (WestleyK) [reply CHANGELOG.md:21]: Uh yeah, right! Its fine as it is

**Review by WestleyK — APPROVED** [review]
Looks good enough to me :shipit:

**Pull request merged**